Portal/Views/Shared/_Layout.cshtml

```diff
@@ -1,4 +1,11 @@
-﻿<!DOCTYPE html>
+﻿@inject Microsoft.AspNetCore.Http.IHttpContextAccessor HttpContextAccessor
+
+@{
+    var httpContext = HttpContextAccessor?.HttpContext ?? new DefaultHttpContext();
+    var (oppositeCultureValue, oppositeCultureName) = CultureHelper.GetCultureInfo(httpContext);
```

Comment:
If culture helper was a class instead of a static, the HttpContextAccessor could be injected into it. Then this class can be injected into the view. I would recommend this method.